extension/src/panel/App.scss

```diff
@@ -250,8 +254,13 @@ a {
   // height: 100%;
 
   &__title {
+    
     margin-bottom: 25px;
   }
+
+  &__H1LineHeight {
```

Comment:
`__header`

Reply:
alternatively change
```
h1 {

}
```

Reply:
To align it correctly, I had to set a line-height for the second title element. Maybe this name of class is better?
I also changed the html structure so that H1 doesn't repeat itself.  
Resolved in bcc8d051b901e96d1dacb999009a669ac43837b5